src/docs/background/09-how-to-launch/index.md

```diff
@@ -29,12 +29,13 @@ to do that now.
 ## 1. Getting access to Flex
 
 If you're reading this article, the chances are that you already have
-access to Flex. This means that you've received the credentials to log
-in to [Flex Console](https://flex-console.sharetribe.com) (the "admin
-section" of your Flex marketplace). If you don't have access yet, you
-can request access by
-[filling out this form](https://sharetribe.typeform.com/to/BI9M4O). If
-you have gotten access to Flex but have forgotten your credentials,
+access a Flex marketplace. This means that you have an account in
+[Flex Console](https://flex-console.sharetribe.com) (the "admin section"
+of your Flex marketplace). If you don't have an account yet, you can
+create one in
+[the Sharetribe Flex website](https://www.sharetribe.com/products/flex/).
```

Comment:
We can also point directly to the Flex creation wizard in the modal with a hash URL. Not sure if it will change still as things are not fully finished, but at the moment the URL would be https://www.sharetribe.com/#start-building-with-flex